mat64/cblas_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build cblas
+
+package mat64
+
```

Comment:
If you want to test with cblas, use `go test -tags cblas`. This does not happen on travis. If you do this locally, it will fail at the moment - we need to harmonise panic values between the two implementations. My proposal would be to drop the c from the beginning of the cblas panics and use those in both - the reasoning being that those messages are more informative (they give a stringer hint to which parameter was incorrect).